services/waifu.ts

```diff
@@ -87,6 +93,20 @@ export const doWaifu = async (body: DoRequest, campaign: string) => {
   return value
 }
 
+export const doSubmitDrop = async (
+  body: DoSubmitDropRequest,
+  campaign: string,
+) => {
+  const value = await api<DoResponse>(`do/${campaign}`, {
```

Comment:
For testing you can use `claimDropItem` but you need to supply image and construct metadata